tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -147,6 +147,34 @@ func createTempFile() string {
 	return fileName
 }
 
+// executing test
+func executeTest(flags []string, m *testing.M) (successCode int, err error) {
+	for i := 0; i < len(flags); i++ {
+		if err = mountGcsfuse(flags[i]); err != nil {
+			log.Printf("mountGcsfuse: %v\n", err)
+			return
+		}
+
+		log.Printf("Test log: %s\n", logFile)
+
+		// Creating a temporary directory to store files
+		// to be used for testing.
+		tmpDir, err = os.MkdirTemp(mntDir, "tmpDir")
+		if err != nil {
+			logAndExit(fmt.Sprintf("Mkdir at %q: %v", mntDir, err))
```

Comment:
same again. not able to follow. Why is logAndExit here and just return in line 155

Reply:
fixed